aviary/interface/methods_for_level2.py

```diff
@@ -609,7 +610,12 @@ def _add_two_dof_takeoff_systems(self):
                 Mission.Design.CRUISE_ALTITUDE, ])
         add_opts2vals(self.model, OptionsToValues, self.aviary_inputs)
 
-        self._add_fuel_reserve_component(post_mission=False)
+        if self.analysis_scheme is AnalysisScheme.SHOOTING:
+            self._add_fuel_reserve_component(post_mission=False)
+        #     add_descent_estimation_as_submodel(
```

Comment:
This is commented out in 2 places. Experimental?

Reply:
experimental that will be finished in a follow on, I didn't want to hold this up since we decided to split the collocation and shooting reserves into separate efforts.